x-pack/plugins/secops/public/components/page.tsx

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiPage } from '@elastic/eui';
+import styled from 'styled-components';
```

Comment:
Since we're using:
[styled-components](https://www.styled-components.com/)

We probably should be adding the linter as soon as we can to catch errors, inconsistencies, type issues, etc...

```
https://github.com/stylelint/stylelint
https://github.com/styled-components/stylelint-processor-styled-components
https://github.com/styled-components/stylelint-processor-styled-components
```
They look pretty standard for people using `styled-components` to add them.

I added a separate ticket for that. No changes requested:
https://github.com/elastic/ingest-dev/issues/70